trieste/models/gpflow/models.py

```diff
@@ -1033,17 +1123,6 @@ def _compute_kernel_blocks(
     Note that `num_latents` is only used when we use a single kernel for a multi-output model.
     """
 
-    if type(inducing_variable) in [gpflow.inducing_variables.SharedIndependentInducingVariables]:
-        inducing_points = inducing_variable.inducing_variable.Z
-    elif type(inducing_variable) in [
-        gpflow.inducing_variables.SeparateIndependentInducingVariables
-    ]:
-        inducing_points = [
-            inducing_variable.Z for inducing_variable in inducing_variable.inducing_variables
-        ]
-    else:
-        inducing_points = inducing_variable.Z
-
     if type(kernel) in [gpflow.kernels.SharedIndependent, gpflow.kernels.SeparateIndependent]:
```

Comment:
Don't fix this line (type in > isinstance) as I'm about to fix it in a separate PR :-)